index.js

```diff
@@ -95,6 +95,9 @@ module.exports = {
   format: function(text, opts) {
     return formatWithShebang(text, normalizeOptions(opts));
   },
+  check: function(text, opts) {
+    return this.format(text, opts) === text;
```

Comment:
It seems weird that if it format throws (syntax error on the input), the check function would throw instead of returning false.